docs/content/concepts/graphql-rpc.mdx

```diff
@@ -15,6 +15,14 @@ Jump to the following sections for more details:
 
 For more details on GraphQL fundamentals, see the introductory documentation from [GraphQL](https://graphql.org/learn/) and [GitHub](https://docs.github.com/en/graphql/guides/introduction-to-graphql).
 
+:::info
```

Comment:
We should put this in a snippet then use {@include: path} so that any updates only have to be made once.

https://github.com/MystenLabs/sui/blob/main/docs/content/references/contribute/mdx-components.mdx#snippets